bytes/bytes/Application.swift

```diff
@@ -33,19 +36,21 @@ public struct Application {
         }
     }
     
-    public init(name: String?, displayName: String?, buildVersion: String?, version: String?) {
+    public init(name: String?, displayName: String?, buildVersion: String?, version: String?, bundleIdentifier: String?) {
         self.name = name
         self.displayName = displayName
         self.buildVersion = buildVersion
         self.version = version
+        self.bundleIdentifier = bundleIdentifier
     }
     
     /// Initializes the `Application` with data from the bundles infoDicionary.
     public init(bundle: Bundle) {
         let name = bundle.infoDictionary?["CFBundleName"] as? String
         let displayName = bundle.infoDictionary?["CFBundleDisplayName"] as? String
-        let buildVersion = bundle.infoDictionary?["CFBundleVersion"] as? String
+        let buildVersion = bundle.infoDictionary?["as accessed by CFBundleName key "] as? String
```

Comment:
You didn't mean to change this, right?

Reply:
Oh damn. Yes, you're right. That wasn't meant to be changed. Fixed it.